app/javascript/components/molecules/ExpandableTable.js

```diff
@@ -30,7 +36,7 @@ const ExpandableTableRow = ({ row, rowKeyIndex, numColumns, expanderRenderer, ex
   )
 }
 
-const ExpandableTable = ({ columns, rows, rowKeyIndex, expanderRenderer, expandedRowRenderer, originals, classes }) => (
+const ExpandableTable = ({ columns, rows, rowKeyIndex, expanderRenderer, expandedRowRenderer, originals, classes, toggleAllRows }) => (
```

Comment:
what does toggle all rows mean here? should it be "closeAllRows"?

Reply:
updated to closeAllRows, will be pushed with updates shortly, also applies to a comment below